rust/benches/benches/models/utils.rs

```diff
@@ -0,0 +1,39 @@
+pub mod vector_or_model {
+    use xaynet_core::mask::{FromPrimitives, Model};
+
+    fn make_vector(bytes_size: usize) -> Vec<i32> {
+        // 1 i32 -> 4 bytes
+        let n_elements = bytes_size / 4;
+        let vector = vec![0_i32; n_elements];
+        vector
+    }
```

Comment:
it's idiomatic rust to just return the result without assigning it to a variable first, not sure why clippy didn't complain here because it usually does. for example like:
```rust
    fn make_vector(bytes_size: usize) -> Vec<i32> {
        // 1 i32 -> 4 bytes
        let n_elements = bytes_size / 4;
        vec![0_i32; n_elements]
    }
```

i would even go one step further and write this as the following, but this is of course personal taste:
```rust
    fn make_vector(bytes_size: usize) -> Vec<i32> {
        // 1 i32 -> 4 bytes
        vec![0_i32; bytes_size / 4]
    }
```

Reply:
yeh i need to get used to this, i usually prefer 1 operation per line. totally happy to go the rust way in the first suggestion, for the second i guess it's case by case